zipkin-query-service/src/main/scala/com/twitter/zipkin/builder/QueryServiceBuilder.scala

```diff
@@ -31,23 +26,24 @@ case class QueryServiceBuilder(override val defaultFinatraHttpPort: String = "0.
                                spanStore: SpanStore,
                                dependencies: DependencyStore = new NullDependencyStore,
                                override val defaultHttpServerName: String = "zipkin-query"
-                                ) extends ZipkinQueryServer(spanStore, dependencies) with
-                                          Builder[RuntimeEnvironment => ListeningServer] {
+                                ) extends ZipkinQueryServer(spanStore, dependencies) {
 
-  override def apply() = (runtime: RuntimeEnvironment) => {
-    LoggerFactory.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME)
-      .asInstanceOf[Logger].setLevel(Level.toLevel(logLevel))
-
-    /** If the span transport is set, trace accordingly, or disable tracing */
+  /** If the span transport is set, trace accordingly, or disable tracing. */
+  def premain {
```

Comment:
@yurishkuro this was a bug!! maybe related to your inability to see spans. pushing a fix now.